lib/lambda_ethereum_consensus/cli.ex

```diff
@@ -24,11 +24,9 @@ defmodule LambdaEthereumConsensus.Cli do
   end
 
   defp init_engine_api_config(_endpoint, nil) do
-    Logger.error(
+    Logger.warning(
       "No jwt file provided. Please specify the path to fetch it from via the --execution-jwt flag."
     )
-
-    System.stop(1)
```

Comment:
we still need to store the endpoint in config here, it needs a bigger refactor.

Besides that, Im fine with the change

Reply:
Done